benchmarks/benchmark.py

```diff
@@ -472,7 +474,12 @@ def modify_config_props_for_mms(pargs):
 
     modify_config_props_for_mms(pargs)
 
-    if pargs.all:
+    if pargs.model:
+        benchmark_model = pargs.model[0].lower()
+        for benchmark_name in ['latency', 'throughput']:
```

Comment:
Define the set of benchmarks as a constant for consistency

Reply:
Done.